src/modules/control.cpp

```diff
@@ -65,14 +122,33 @@ static PT_THREAD(controlThreadMain(struct pt* pt))
     } else if (control.state == CONTROL_BEGIN_EXHALATION) {
       // Since exhalation is not dependent on the bag, allow the bag to decompress with the same parameters as compression
       // In order to time exhalation, set a timer to time the exhalation cycle
-      // TODO: consider if patient takes breathe before motor has completely moved back
-      motorHalBegin(MOTOR_HAL_DIRECTION_EXHALATION, motorCompressionDistance, motorCompressionDuration);
-      timerHalBegin(&controlTimer, 1 MSEC); // TODO: calculate this value from the breathe parameters
+      // TODO: Andrew, thoughts?; also, if this is a motor home, maybe need different HAL function for it
+      motorHalBegin(MOTOR_HAL_DIRECTION_EXHALATION,
+                    totalMotorCompressionDistance,
+                    totalMotorCompressionDuration);
```

Comment:
I don't think that it's necessary to use totoalMotorCompressionDuration here, but I think it's a moot point as we are moving to a different method anyways (speed control instead of duration command).